VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -21,22 +21,26 @@
     private Observer mObserver;
 
     IEdxEnvironment environment;
+    UserAPI api;
```

Comment:
Actually, I guess the `environment` variable could also be injected along with it.